ckcp/openshift_dev_setup.sh

```diff
@@ -309,8 +309,25 @@ install_openshift_pipeline() {
   APP="argocd"
 
   echo -n "  - openshift-pipeline application: "
+
+  # Generate postgres secret
+  kubectl create secret generic tekton-results-postgres --namespace="openshift-pipelines" --from-literal=POSTGRES_USER=postgres --from-literal=POSTGRES_PASSWORD=$(openssl rand -base64 20) --dry-run=client -o yaml | kubectl apply -f - >/dev/null 2>&1
```

Comment:
This script stops at this point -
```bash
+ echo -n '  - openshift-pipeline application: '
  - openshift-pipeline application: + kubectl apply -f -
++ openssl rand -base64 20
+ kubectl create secret generic tekton-results-postgres --namespace=openshift-pipelines --from-literal=POSTGRES_USER=postgres --from-literal=POSTGRES_PASSWORD=x9+6KZHuEsmghDHq0Z5EPYczrbs= --dry-run=client -o yaml
```
To me, the possible cause seems to be the fact that without `Openshift-Pipelines` operator, there's no namespace `--namespace=openshift-pipelines` which would be created later with `tektoncd` argo-app [here](https://github.com/openshift-pipelines/pipelines-service/pull/139/files#diff-0ccaa908000d6cb2cc1643e32a307cf89a8cbf5bd36ceab4557ef50817def2e1R329)
Would it be a good idea to create a different function to configure tekton-results and handle this chronology?

Reply:
Updated to install from a separate function